### ISSUE TITLE
Shipping Address and Items are not showing in eWay shared payment page

### ISSUE THREAD

**Issue by A5hpat** [eWAYPayment/eway-rapid-net#2]
Hi
In ResponsiveShared, Shipping Address and Items are not showing in eWay shared payment page.
![Image of issue](http://www.ashishblog.com/wp-content/uploads/2015/10/Capture2.png)
I debug code and see that request (eWAY.Rapid.Models.Transaction) to CreateAccessCodeSharedRequest (eWAY.Rapid.Internals.Request.CreateAccessCodeSharedRequest ) mapping missing Shipping Address and Items objects.

see below code RapidClient.cs

``` javascript
 private CreateTransactionResponse CreateTransaction<TRequest, TResponse>(Func<TRequest, TResponse> invoker, Transaction transaction)
        {
            var request = _mappingService.Map<Transaction, TRequest>(transaction);
            var response = invoker(request);
            return _mappingService.Map<TResponse, CreateTransactionResponse>(response);
        }
```

Shipping Address and Items objects are in transaction but missing into request when it map to TRequest (CreateAccessCodeSharedRequest)

``` javascript
var request = _mappingService.Map<Transaction, TRequest>(transaction);
```

**Comment by incarnate** [comment]
Hi Ashish,

Thanks for pointing out this issue and doing some debugging. We've got a fix in the works, just checking to make sure we haven't missed these fields in other requests as well.

Cheers,
John

**Comment by incarnate** [comment]
We've just release version 1.1.2 of the eWAY .NET SDK to [NuGet](https://www.nuget.org/packages/eWAY.Rapid/) which resolves the issue with passing shipping and item details.

**Comment by A5hpat** [comment]
Hi John,

Thanks for quick fixed. 
also those fields are missing in QueryTransactionResponse.

Thanks,
Ashish

**Comment by incarnate** [comment]
Thanks for picking that up as well Ashish, we've just released v1.1.3 which adds the shipping and invoice details fields to the query transaction response.

Cheers,
John
